src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -566,10 +633,10 @@ def get_load_balancer_sku(
         # set default value
         read_from_mc = False
         if value_obtained_from_mc is not None:
-            load_balancer_sku = value_obtained_from_mc
+            load_balancer_sku = safe_lower(value_obtained_from_mc)
```

Comment:
unit test for the lower case.

Reply:
Sure, changed in commit [5150709](https://github.com/Azure/azure-cli/pull/19458/commits/5150709ce10d95bd7d180d31b1be468e84583b06).